prestoadmin/util/base_config.py

```diff
@@ -103,6 +140,24 @@ class BaseConfig(object):
     def __init__(self, config_path, config_items):
         self.config_path = config_path
         self.config_items = config_items
+        self.config = {}
+
+    def __getitem__(self, key):
```

Comment:
Act like a dict so we can set env.conf for standalone configs as well as yarn_slider configs.